packages/engine-server/src/metadata/service.ts

```diff
@@ -233,6 +237,10 @@ export class MetadataService {
     return this.getMeta().priorTools;
   }
 
+  get RecentWorkspaces(): string[] | undefined {
```

Comment:
we have a mix of `get {Prop}` and `get{Prop}` calls here (and everywhere). can we have an async to standardize? ()